src/performance_profiles.jl

```diff
@@ -152,3 +154,48 @@ function performance_profile(
     kwargs...,
   )
 end
+
+"""
+  export_performance_profile(T; solver_names = [], logscale::Bool = true, sampletol=0.0, drawtol=0.0,)
```

Comment:
this still doesn't match the function's signature below.

Reply:
It's solved.